src/program/lwaftr/tests/subcommands/run_nohw_test.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test the "snabb lwaftr run_nohw" subcommand.
+"""
+
+import unittest
+
+from random import randint
+from subprocess import call, check_call
+from test_env import DATA_DIR, SNABB_CMD, BaseTestCase
+
+class TestRun(BaseTestCase):
+
+    program = [
+        str(SNABB_CMD), 'lwaftr', 'run_nohw',
+    ]
+    cmd_args = {
```

Comment:
I like this separation between arguments, that have to be in the right order, and options, that have no such constraint, and can be better represented by a dictionary. I would rename `program` to `cmd_args` and `cmd_args` to `cmd_options`, though,